communitiesuk/funding-service-pre-award
# FLS-827-hsra-configs

https://mhclgdigital.atlassian.net/browse/FLS-827

### Change description
- HSRA VR and RP configured in Apply and Assess
- Old HSRA R1 removed
- tests updated
- hsra seed data added
- added seed_assessment_store script to add scoring systems in DB
- seed_assessment_store script documentation added



### How to test
- Verify HSRA round VR and RP are available on apply and assess and are working as expected


### Screenshots of UI changes (if applicable)
![image](https://github.com/user-attachments/assets/ca6e9575-e1e3-4e0f-ab27-5ef2dff588aa)


![image](https://github.com/user-attachments/assets/ab95a21f-b965-4b80-b2b1-babb94273a25)


![image](https://github.com/user-attachments/assets/97ac071d-a16d-40e2-ab90-69713f4f022b)
